winter/pagination/page/page_processor.py

```diff
@@ -0,0 +1,23 @@
+import typing
+
+from rest_framework.request import Request as DRFRequest
+
+from .page import Page
+from ..utils import get_next_page_url
+from ..utils import get_previous_page_url
+from ...output_processor import IOutputProcessor
+
+
+class PageProcessor(IOutputProcessor):
+
+    def process_output(self, output: Page, request: DRFRequest) -> typing.Dict:
```

Comment:
sometimes you use namespaces, like typing.Dict. Sometimes not (like ABC). Is there a rule here?

Reply:
We need to have the rule, but now we do not have